invenio_previewer/templates/semantic-ui/invenio_previewer/txt.html

```diff
@@ -10,11 +10,10 @@
 {%- extends config.PREVIEWER_ABSTRACT_TEMPLATE %}
 
 {% block panel %}
-<div class="ui container">
-    <div class="ui grid column">
-        <div class="column">
-            <pre>{{ content }}</pre>
-        </div>
-    </div>
+<div id="text-previewer">
```

Comment:
For the future: maybe we can simply use the autogenerated classes? See [here](https://github.com/inveniosoftware/invenio-theme/blob/d44a5b72eeb0716ec8081731df552fafa8f8a3de/invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides#L358)
```suggestion
<div class="rel-p-1">
```